### DISCUSSION TITLE
Adding text along a path not working anymore?

### DISCUSSION THREAD

**Discussion by NATateo** [cduck/drawSvg#128] [General]
No matter what I try, it seems I cannot create a text along a path anymore.
Even if I try the example code on the drawsvg homepage (drawing 'path text' and 'multi line text' along the green line) I only get those words (on a single line, so multi line isn't working either) at the center of the drawing.

Is it just me? No matter what I tried it doesn't seem to work. What can I try?

**Comment by cduck** [comment]
Hi @NATateo, thank you for asking.  When you ask why something doesn't work anymore, it is very helpful to me if you can tell me which version of drawsvg worked in the pase and which version of drawsvg doesn't work now.  This can help me determine what broke and how to fix it.

Path text and multi-line text works for me in the latest version (drawsvg==2.3.0) so I can't help you until you provide more details.

- Find the version by running the command `python -m pip show drawsvg`
- Upgrade to the latest version with the command `python -m pip install -U drawsvg` or `python -m pip install drawsvg==2.3.0`
- Try the first README example again.
- View https://github.com/cduck/drawsvg/blob/master/examples/example1.svg in an up-to-date web browser (Firefox, Safari, or Chrome).

**Reply by NATateo** [reply]
You're 100% right, I'm sorry I forgot to add that I already updated just a few days ago (I'm using Spyder via Anaconda) and I have the latest version, 2.3.0.

In the meantime I solved the issue: there's nothing wrong with the code, it's Spyder which seems incapable of showing correctly the generated drawing in the console. If I save the output in a file and I open it with an external app, the text is right where it's supposed to be.
Unfortunately that makes almost impossible to use it, because I have to add many different texts and check the proper positioning of each would take too long.
Is there another way to show a svg drawing directly in the console?

Thank you for your support and for the great package, I love drawsvg.

edit: nevermind, now that I know the problem I easily found a workaround: if I call drawing.rasterize() in the console the output is correct, so I can use that to have a quick view of the result.
